prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -96,13 +139,25 @@ def build_plant(bus, substations):
     # Initial loading
     generators = load.get_eia_form_860(const.blob_paths["eia_form860_2019_generator"])
     plants = load.get_eia_form_860(const.blob_paths["eia_form860_2019_plant"])
+    print("Fetching EPA data... (this may take several minutes)")
+    epa_ampd = load.get_epa_ampd(const.blob_paths["epa_ampd"])
+    crosswalk = load.get_eia_epa_crosswalk(const.eia_epa_crosswalk_path)
 
     # Data interpretation
     plants = plants.set_index("Plant Code")
     plants["Latitude"] = plants["Latitude"].map(floatify)
     plants["Longitude"] = plants["Longitude"].map(floatify)
     for col in ["Summer Capacity (MW)", "Winter Capacity (MW)", "Minimum Load (MW)"]:
         generators[col] = generators[col].map(floatify)
+    epa_ampd = epa_ampd.astype({"UNITID": "string"})
```

Comment:
Should we move this in `get_epa_ampd` function in the `load` module?

Reply:
Sure.

Reply:
Done.